Sources/Confidence/DefaultStorage.swift

```diff
@@ -35,6 +35,7 @@ class DefaultStorage: Storage {
         do {
             return try JSONDecoder().decode(T.self, from: data)
         } catch {
+            try! clear()
```

Comment:
Should we log errors (with our logger) and return FlagResolution.EMPTY to prevent crashes?